app/pages/articles/[articleId].tsx

```diff
@@ -107,7 +108,7 @@ const ArticleDetails = (props) => {
         </div>
         <div className="article__author text-base text-center m-2 max-w-2xl text-gray-dark dark:text-gray-light">
           <FaCrown className="inline m-2" />
-          {article.authorString}
+          {article.authorString} ({article.publishedYear})
```

Comment:
` {article.authorString} {publishedYear}` will work and print out the string with the parenthesis already? Also, in the case there is no published year the filed will be blank, as opposed to showing as "()"

Reply:
ooh! that would be great! can you change it!